superset-frontend/src/setup/setupApp.ts

```diff
@@ -67,7 +67,7 @@ export default function setupApp() {
     ) {
       ev.preventDefault();
       SupersetClient.get({
-        endpoint: ev.currentTarget.href,
+        url: ev.currentTarget.href,
```

Comment:
Are we sure that the url will always be a full url instead of a relative one?

Reply:
Even if it's not, SupersetClient should still be able to handle it. `fetch` does accept relative URLs.